docs/source-pytorch/advanced/training_tricks.rst

```diff
@@ -254,6 +278,29 @@ via ``self.learning_rate`` or ``self.lr``.
 
     trainer.tune(model)
 
+You can also customize the :class:`~pytorch_lightning.callbacks.lr_finder.LearningRateFinder` callback to run at different epochs. This feature is useful while fine-tuning models.
```

Comment:
I think this deserves its own section title. For example: "Customizing the lr finder". 
Maybe put it after this section, above "Advanced GPU Optimizations"?

Reply:
good idea.. done!